tkltest/util/unit/build_util.py

```diff
@@ -433,6 +433,7 @@ def __build_maven(classpath_list, app_name, monolith_app_paths, test_root_dir, t
                         line('artifactId', 'maven-surefire-plugin')
                         line('version', constants.MAVEN_SURFIRE_VERSION)
                         with tag('configuration'):
+                            line('testFailureIgnore', 'true')
```

Comment:
@haim-kermany this ignores failures, but we should also ignore test errors. Will this solution work if a test ends in error, for example throws a `NullPointerException`?

Reply:
yes, it apply also for errors, I checked